CMakeLists.txt

```diff
@@ -127,10 +129,13 @@ if (BUILD_STATIC_RELEASE)
   set(Boost_USE_STATIC_RUNTIME OFF)
   # Set the static variable
   set(P4C_STATIC_BUILD STATIC)
+  # TODO: We can not use -static here because of compilation and portability problems:
+  # TODO: Keep track of https://stackoverflow.com/questions/3430400/linux-static-linking-is-dead
+  # and see whether static linking improves on Linux.
   # Do not bring in dynamic libstdcc and libgcc
-  set(CMAKE_EXE_LINKER_FLAGS "-static -static-libgcc -static-libstdc++ -Wl,-z,muldefs")
+  set(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++ -Wl,-z,muldefs")
```

Comment:
It would still make sense to include the previous value of `CMAKE_EXE_LINKER_FLAGS` I think. The user should have ability to add to them.